global-privacy-control/tests.json

```diff
@@ -22,6 +22,33 @@
                 "expectGPCHeader": false,
                 "exceptPlatforms": []
             },
+            {
+                "name": "Header not injected on platforms using gpcHeaderEnabledSites - domain is excluded from gpcHeaderEnabledSites",
+                "siteURL": "http://anotherexample.com",
+                "requestURL": "http://anotherexample.com",
+                "requestType": "main_frame",
+                "gpcUserSettingOn": true,
+                "expectGPCHeader": false,
+                "exceptPlatforms": [
+                    "web-extension",
+                    "safari-extension",
+                    "windows-browser",
+                    "android-browser"
```

Comment:
Android is using gpcHeaderEnabledSites, but AFAIK they are using it a bit differently than apple platforms, so it makes sense to exclude it here 👍

Reply:
Cool, I checked with Marcos before doing this, they do use it, but not in a way it would work for these tests.